packages/mantine/src/theme/Theme.tsx

```diff
@@ -111,36 +112,56 @@ export const plasmaTheme: MantineThemeOverride = createTheme({
             },
             classNames: {title: AlertClasses.title},
         }),
-        Title: Title.extend({
-            classNames: {root: TitleClasses.root},
+        Anchor: Anchor.extend({
+            defaultProps: {
+                color: 'action.6',
+            },
+            classNames: {root: AnchorClasses.root},
         }),
-        Text: Text.extend({
-            classNames: TextClasses,
-            vars: TextVars,
+        AppShellNavbar: AppShellNavbar.extend({
+            classNames: {navbar: AppShellNavBarClasses.navbar},
+        }),
+        Badge: Badge.extend({
+            classNames: {root: BadgeClasses.root},
+            defaultProps: {
+                variant: 'light',
+            },
         }),
         Button: Button.extend({
             classNames: ButtonClasses,
         }),
-        LoadingOverlay: LoadingOverlay.extend({
-            defaultProps: {zIndex: 1400},
+        Checkbox: Checkbox.extend({
+            defaultProps: {
+                radius: 'sm',
+            },
+            classNames: {label: CheckboxClasses.label, input: CheckboxClasses.input},
         }),
         CloseButton: CloseButton.extend({
             defaultProps: {
                 icon: <CrossSize16Px height={16} aria-label="close" />,
             },
         }),
-        Modal: Modal.extend({
-            classNames: ModalClasses,
+        ColorSwatch: ColorSwatch.extend({
+            defaultProps: {
+                size: 8,
+                withShadow: false,
+            },
         }),
-        ModalRoot: Modal.Root.extend({
+        Combobox: Combobox.extend({
+            classNames: {option: SelectClasses.option, search: ComboboxClasses.search},
+        }),
+        ComboboxSearch: ComboboxSearch.extend({
             defaultProps: {
-                padding: 'lg',
+                placeholder: 'Search...',
```

Comment:
I've seen somewhere someone from the doc recommending to remove the dots here, lets remove them from the default placeholder.

```suggestion
                placeholder: 'Search',
```

Reply:
yep I saw it too, I'll remove it